### PR TITLE
Updates to Denver hometown edit

### PR DESCRIPTION
**Pull request by jdavis1998**
Updates to sites and added emoji

### CODE REVIEW

**Comment by githubteacher** [comment]
What is your favorite pizza place?

File: jdavis1998/jdavis1998-Denver.md [hunk 1]
```diff
@@ -2,10 +2,12 @@
 
 #### Restaurants
 
-- El Tequileno 
+- El Tequileno
 - Yolanda's Tacos
 - Welton Street Cafe
```
Comment (doppeldimmykumont) [inline jdavis1998/jdavis1998-Denver.md:7]: I don't live too far from there!

File: jdavis1998/jdavis1998-Denver.md [hunk 2]
```diff
@@ -2,10 +2,12 @@
...
+- Sushi Hai :heart:
 
 #### Things to do
 - Botanical Gardens
```
Comment (doppeldimmykumont) [inline jdavis1998/jdavis1998-Denver.md:11]: My favorite place in Denver!

File: kevinpmason-Denver.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+## Things to do in Denver when you're alive
+
+#### Restaurants
+- Bluebonnet Cafe
+- Casa Bonita
```
Comment (doppeldimmykumont) [inline kevinpmason-Denver.md:5]: HAHAHAHA!

File: jdavis1998/jdavis1998-Denver.md (hunk 1 above)
Comment (jdavis1998) [inline jdavis1998/jdavis1998-Denver.md:7]: Pizza place would be "Atomic Cowboy"

File: jdavis1998/jdavis1998-Denver.md (hunk 2 above)
Comment (jdavis1998) [inline jdavis1998/jdavis1998-Denver.md:11]: Only sopapillas are good at Casa Bonita

**Pull request merged**